src/test/java/org/entando/plugins/pda/pam/service/process/KieProcessFormServiceTest.java

```diff
@@ -0,0 +1,187 @@
+package org.entando.plugins.pda.pam.service.process;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.drools.core.io.impl.ClassPathResource;
+import org.entando.plugins.pda.core.model.form.Form;
+import org.entando.plugins.pda.core.model.form.FormField;
+import org.entando.plugins.pda.core.model.form.FormFieldInteger;
+import org.entando.plugins.pda.core.model.form.FormFieldText;
+import org.entando.plugins.pda.core.model.form.FormFieldType;
+import org.junit.Assert;
+import org.junit.Test;
+
+@SuppressWarnings("PMD.ExcessiveMethodLength")
+public class KieProcessFormServiceTest {
+
+    @Test
+    public void deserializeProcessFormJson() {
```

Comment:
This tests belongs to `KieProcessFormDeserializerTest`.
Check `KieTaskServiceTest` to see the kind of unit tests that should be implemented on this class.